tools/run_gpu_tests.sh

```diff
@@ -9,5 +9,5 @@ if [ "$DOCKER_BUILDKIT" == "" ]; then
   export DOCKER_BUILDKIT=1
 fi
 
-docker build -f tools/docker/gpu_tests.Dockerfile -t tfa_gpu_tests ./
+docker build -f tools/docker/gpu_tests.Dockerfile --target gpu_tests -t tfa_gpu_tests ./
 docker run --rm -t --runtime=nvidia tfa_gpu_tests
```

Comment:
@gabrieldemarmiesse Do you want to run this also on `interactive_dev`? Or do you want to `docker run` something different there?

Reply:
@gabrieldemarmiesse An alternative is to execute `bazel build --nobuild -- //tensorflow_addons/...` if it is enough for testing `interactive_dev`.